### DISCUSSION TITLE
Codeforce导入报错Forbidden

### DISCUSSION THREAD

**Discussion by yyym-y** [Liu233w/acm-statistics#3023] [Bug reports]
![image](https://github.com/Liu233w/acm-statistics/assets/133961507/b43d69ac-7731-4527-a38c-90c8cc847c5e)

**Comment by Liu233w** [comment]
![CleanShot 2024-07-01 at 03 17 53@2x](https://github.com/Liu233w/acm-statistics/assets/16333687/c2c95333-bd00-4298-8843-1cd779aab5fb)

我刚刚试了一下有结果啊
